### PR TITLE
Main synced to 111

### PR DESCRIPTION
**Pull request by filipi87**
Latest version of `react-native-webrtc`, which is using webrtc 111 and software encode/decode for Android.

Android codecs:
In hardware: H.264 (high, baseline)
In software: VP8, VP9, AV1

Bugs that have been fixed with this version:
  - Android is respecting the simulcast, sending multiple layers. We are using VP8 as codec.
  - Android is respecting when just changing the `maxQuality`, to send the right layer, respecting also the resolution and FPS, which was not respecting before.
  - No changes on iOS. Layer 01 is always using  `scaleResolutionDownBy` 1 and `frameRate` 30.   And Layer 00 is always using  `scaleResolutionDownBy` 2 and `frameRate` 30. The bitrate is respected by both layers.

### CODE REVIEW

**Comment by filipi87** [comment]
I still intend to profile android CPU usage, and compare with the previous version.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@daily-co/react-native-webrtc",
-  "version": "106.0.6-daily.1",
+  "version": "111.0.0-daily.1",
```
Comment (kompfner) [inline package.json:3]: at some point we switched from being 0-indexed (starting at `-daily.0`) to being 1-indexed?

**Review by kompfner — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (filipi87) [reply package.json:3]: Maybe I did that without intention on the previous release, but I can fix that and return to be `0`.

**Review by markbackman — APPROVED** [review]
Tested iOS and Android on M111 and saw the expected results in terms of simulcast layers. Approving for that reason.

The simulcast settings aren't where we want them to be, so we should tackle that in another task: https://linear.app/dailyco/issue/CSDK-1036/update-simulcast-settings-for-react-native.

**Comment by filipi87** [comment]
The amount of memory that we are using has not changed that much, but the CPU, we are consuming almost 02 times more than before.

One thing to consider, is that now we are sending 02 layers, before, even if we had the simulcast defined, android was sending only 01 layer.

Old version of react-native-webrtc, using hardware codecs:
Screenshot 2023-04-20 at 15 14 43

New version, using software codecs:
Screenshot 2023-04-20 at 15 22 27

**Pull request merged**